### PR TITLE
disabling few signal handlers from node

### PR DESCRIPTION
**Pull request by madanagopalt**

### CODE REVIEW

File: examples/pxScene2d/external/libnode-v6.9.0/deps/openssl/openssl/crypto/armcap.c [hunk 1]
```diff
@@ -100,21 +104,29 @@ void OPENSSL_cpuid_setup(void)
         return;
     }
 
+/* MODIFIED CODE BEGIN */
+/*
     sigfillset(&all_masked);
     sigdelset(&all_masked, SIGILL);
-    sigdelset(&all_masked, SIGTRAP);
```
Comment (FitzerIRL) [inline examples/pxScene2d/external/libnode-v6.9.0/deps/openssl/openssl/crypto/armcap.c]: Might be better to comment out - versus remove entirely.

File: examples/pxScene2d/external/libnode-v6.9.0/deps/openssl/openssl/crypto/armcap.c [hunk 2]
```diff
@@ -100,21 +104,29 @@ void OPENSSL_cpuid_setup(void)
...
     sigdelset(&all_masked, SIGFPE);
-    sigdelset(&all_masked, SIGBUS);
```
Comment (FitzerIRL) [inline examples/pxScene2d/external/libnode-v6.9.0/deps/openssl/openssl/crypto/armcap.c]: Might be better to comment out - versus remove entirely.

File: examples/pxScene2d/external/libnode-v6.9.0/deps/v8/include/v8.h [hunk 3]
```diff
@@ -791,6 +791,7 @@ template <class T, class M> class Persistent : public PersistentBase<T> {
   template <class S> V8_INLINE Persistent<S>& As() { // NOLINT
     return Persistent<S>::Cast(*this);
   }
+  V8_INLINE T* operator*() const { return this->val_; }
```
Comment (mfiess) [inline examples/pxScene2d/external/libnode-v6.9.0/deps/v8/include/v8.h]: Please add the modified code comments here.

File: examples/pxScene2d/external/libnode-v6.9.0/deps/v8/include/v8.h [hunk 4]
```diff
@@ -800,7 +801,7 @@ template <class T, class M> class Persistent : public PersistentBase<T> {
   template<class F> friend class ReturnValue;
 
   explicit V8_INLINE Persistent(T* that) : PersistentBase<T>(that) {}
-  V8_INLINE T* operator*() const { return this->val_; }
+  //V8_INLINE T* operator*() const { return this->val_; }
```
Comment (mfiess) [inline examples/pxScene2d/external/libnode-v6.9.0/deps/v8/include/v8.h]: Please add the modified code comments here.

File: examples/pxScene2d/external/libnode-v6.9.0/deps/v8/src/js/array.js [hunk 5]
```diff
@@ -520,7 +520,7 @@ function ObservedArrayPush() {
 // length of the array. See ECMA-262, section 15.4.4.7.
 function ArrayPush() {
   CHECK_OBJECT_COERCIBLE(this, "Array.prototype.push");
-
+  console.log("Madana ....................");
```
Comment (mfiess) [inline examples/pxScene2d/external/libnode-v6.9.0/deps/v8/src/js/array.js]: Please remove

**Comment by conniefry** [comment]
Closing this PR without merging.  This can be a much simpler change.

**Pull request closed without merging**